tools/backups2datalad/asyncer.py

```diff
@@ -243,65 +239,41 @@ async def feed_addurl(self) -> None:
                 log.debug("Done feeding URLs to addurl")
 
     async def read_addurl(self) -> None:
-        async with self.post_sender:
-            async with aclosing(
-                aiter(self.addurl)
-            ) as lineiter:  # type: ignore[type-var]
-                async for line in lineiter:
-                    data = json.loads(line)
-                    if "success" not in data:
-                        # Progress message
-                        log.info(
-                            "%s: Downloaded %d / %s bytes (%s)",
-                            data["action"]["file"],
-                            data["byte-progress"],
-                            data.get("total-size", "???"),
-                            data.get("percent-progress", "??.??%"),
-                        )
-                    elif not data["success"]:
-                        log.error(
-                            "%s: download failed:%s",
-                            data["file"],
-                            format_errors(data["error-messages"]),
-                        )
-                        self.in_progress.pop(data["file"])
-                        self.report.failed += 1
-                    else:
-                        path = data["file"]
-                        key = data.get("key")
-                        log.info("%s: Finished downloading (key = %s)", path, key)
-                        self.report.downloaded += 1
-                        dl = self.in_progress.pop(path)
-                        await self.post_sender.send((dl, key))
-            log.debug("Done reading from addurl")
-
-    async def postprocess(self) -> None:
-        async with self.post_receiver:
-            async for dl, key in self.post_receiver:
-                self.tracker.finish_asset(dl.path)
-                if key is not None:
-                    for u in dl.extra_urls or []:
-                        await self.register_url(dl.path, key, u)
-                    annex_hash = key2hash(key)
-                    if dl.sha256_digest != annex_hash:
-                        log.error(
-                            "%s: Hash mismatch!  Dandiarchive reports %s,"
-                            " local file has %s",
-                            dl.path,
-                            dl.sha256_digest,
-                            annex_hash,
-                        )
-                        self.report.hash_mismatches += 1
+        async with aclosing(aiter(self.addurl)) as lineiter:  # type: ignore[type-var]
+            async for line in lineiter:
+                data = json.loads(line)
+                if "success" not in data:
```

Comment:
I think this block would be more robust if checking 
```suggestion
                if "byte-progress" in data:
```
since code block is about reporting progress.